one/src/main.rs

```diff
@@ -220,6 +220,40 @@ impl Daemon {
         };
         debug!("using directory: {}", dir.display());
 
+        let mut bootstrap_addresses = opts.bootstrap_addresses.clone();
+        // Add the default bootstrap peers for Mainnet, TestnetClay, and DevUnstable.
+        // from https://github.com/ceramicnetwork/js-ceramic/blob/develop/packages/ipfs-topology/src/ipfs-topology.ts
+        let bootstrap_addresses = match opts.network {
```

Comment:
We could add this `match` expression as a method on the `Network` type itself. That way its clear the list of bootstrap address is a property of the network selection itself. Thoughts?

Reply:
Moved to Network enum looks cleaner.